config/initializers/content_security_policy.rb

```diff
@@ -4,22 +4,22 @@
 # For further information see the following documentation
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
 
```

Comment:
Won't this prevent `marvinJS` from running?

Reply:
we can whitelist it

Reply:
Cool, we should I guess